shared/js/anim_sync.js

```
@@ -0,0 +1,228 @@
+    }
+
+    // Write metadata key with chunk count
+    const metaKey = animMetaKey(animationId);
+    let metaSuccess = false;
+
+    for (let attempt = 0; attempt < MAX_RETRIES; attempt++) {
+        try {
+            await chrome.storage.sync.set({ [metaKey]: { chunkCount: totalChunks } });
+            metaSuccess = true;
+            break;
+        } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)